app/workers/process_content_change_and_generate_emails_worker.rb

```diff
@@ -5,8 +5,10 @@ class ProcessContentChangeAndGenerateEmailsWorker < ProcessAndGenerateEmailsWork
 
   def perform(content_change_id)
     content_change = ContentChange.find(content_change_id)
+
     return if content_change.processed?
 
+
```

Comment:
Maybe get rid of these changes (or move them to a commit that says why blank lines are being added).

Reply:
I can see that you've removed these lines in a subsequent commit. That's better, but still not as good as not adding them in the first place. I'll forgive you though as you're not using Emacs yet.